phones/tests/mgmt_delete_phone_data_tests.py

```diff
@@ -0,0 +1,272 @@
+import pytest
+
+if settings.PHONES_ENABLED:
+    from ..models import InboundContact, RealPhone, RelayNumber
+
+pytestmark = pytest.mark.skipif(
+    not settings.PHONES_ENABLED, reason="PHONES_ENABLED is False"
+)
+THE_COMMAND = "delete_phone_data"
+
+
+@pytest.fixture(autouse=True)
+def test_settings(settings: SettingsWrapper) -> SettingsWrapper:
+    """Override settings for tests"""
+    settings.SUBSCRIPTIONS_WITH_PHONE = ["PHONE_SUBSCRIPTION"]
+    return settings
+
+
+@pytest.fixture(autouse=True)
+def mock_twilio_client() -> Iterator[Mock]:
+    """Mock PhonesConfig with a mock twilio client"""
+    with patch("phones.apps.PhonesConfig.twilio_client") as mock_twilio_client:
+        yield mock_twilio_client
+
+
+@pytest.fixture
+def phone_user(db: None, test_settings: SettingsWrapper) -> User:
+    """Return a Relay user with phone setup and phone usage."""
+    # Create the user
+    now = datetime.now(tz=timezone.utc)
+    phone_user = baker.make(User, email="phone_user@example.com")
+    phone_user.profile.date_subscribed = now - timedelta(days=15)
+    phone_user.profile.save()
+
+    # Add FxA with phone subscription
+    baker.make(
+        SocialAccount,
+        user=phone_user,
+        provider="fxa",
+        uid="decd14f50fce4865b0fd06afcd9d2c4d",
+        extra_data={
+            "avatar": "http://avatar.example.com/avatar.jpg",
+            "subscriptions": [settings.SUBSCRIPTIONS_WITH_PHONE[0]],
+        },
+    )
+
+    # Add confirmed real phone
+    baker.make(
+        RealPhone,
+        user=phone_user,
+        number="+12005550123",
+        verification_sent_date=now - timedelta(days=14),
+        verified=True,
+    )
+
+    # Add relay phone mask
+    relay_number = baker.make(RelayNumber, user=phone_user, number="+13015550123")
+
+    # Add SMS and Voice contacts
+    baker.make(
+        InboundContact,
+        relay_number=relay_number,
+        inbound_number="+14035550123",
+        last_inbound_type="text",
+        num_texts=1,
+        last_inbound_date=now - timedelta(days=12),
+        last_text_date=now - timedelta(days=12),
+    )
+    baker.make(
+        InboundContact,
+        relay_number=relay_number,
+        inbound_number="+14045550123",
+        last_inbound_type="call",
+        num_calls=1,
+        last_inbound_date=now - timedelta(days=10),
+        last_call_date=now - timedelta(days=10),
+    )
+
+    return phone_user
+
+
+def test_active_user(phone_user: User) -> None:
+    """A user's real phone, relay phone, and inbound contacts are deleted."""
+    assert RealPhone.objects.filter(user=phone_user).exists()
+    relay_number = RelayNumber.objects.get(user=phone_user)
+    assert InboundContact.objects.filter(relay_number=relay_number).count() == 2
+
+    stdout = StringIO()
+    call_command(THE_COMMAND, phone_user.profile.fxa.uid, stdout=stdout)
+
+    expected_stdout = f"""\
+Found a matching user:
+
+* FxA ID: decd14f50fce4865b0fd06afcd9d2c4d
+* User ID: {phone_user.id}
```

Comment:
question (non-blocking): Why reference the `phone_user` object for this value but not the others? E.g., why not this?
```python
* FxA ID: {phone_user.profile.fxa.uid}
* User ID: {phone_user.id}
* Email: {phone_useremail}
* etc.
```

Reply:
because in the test cases, those values do not change, but the user ID changes depending on what tests ran before this one.

Reply:
Got it, thanks for the clarification.